gme/gme.h

```diff
@@ -91,12 +91,18 @@ BLARGG_EXPORT int gme_tell( Music_Emu const* );
 /* Number of samples generated since beginning of track */
 BLARGG_EXPORT int gme_tell_samples( Music_Emu const* );
 
+/* Number of milliseconds played since beginning of track (scaled with tempo) */
+BLARGG_EXPORT int gme_tell_scaled( Music_Emu const* );
```

Comment:
Please add a `since` line in the comment, like: ` * @since 0.6.5`
(see other prototypes with suc a comment)